bap-builder/PackageMode.go

```
@@ -198,33 +200,73 @@
+	handleRemover := bringauto_process.SignalHandlerAddHandler(repo.RestoreAllChanges)
+	if *cmdLine.All {
+		err = buildAllPackages(cmdLine, contextPath, platformString, repo)
+	} else {
+		err = buildSinglePackage(cmdLine, contextPath, platformString, repo)
 	}
-	return buildSinglePackage(cmdLine, contextPath)
+	if err != nil {
+		handleRemover()
+		return err
+	}
+	repo.CommitAllChanges()
```

golangci-lint (1.62.2)

Error return value of `repo.CommitAllChanges` is not checked

(errcheck)